src/app/(contents)/post/register/PostRegister.tsx

```
@@ -0,0 +1,174 @@
+    }
+
+    const uploadMultiFile = async (contentHtml: string, base64Images: string[]): Promise<string> => {
+        try {
+            const response = await axiosInstance.post("/google-drive/upload-multi", { base64Images })
+            const urls: string[] = response.data.data
+
+            urls.forEach((url, idx) => {
+                contentHtml = contentHtml.replace(base64Images[idx], url)
+            })
+
+            return contentHtml
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)